packages/cape/src/methods.ts

```diff
@@ -216,17 +221,26 @@ export abstract class Methods {
       const doc = await this.verifyAttestationDocument(data.attestation_document);
 
       const obj = JSON.parse(new TextDecoder().decode(doc.user_data));
-
-      return '-----BEGIN PUBLIC KEY-----\n' + addNewLines(obj.key) + '\n-----END PUBLIC KEY-----';
+      const keyString = '-----BEGIN PUBLIC KEY-----\n' + addNewLines(obj.key) + '\n-----END PUBLIC KEY-----';
+      this.rsaKeyCache.set(username, keyString);
+      return keyString;
     }
 
     try {
+      // We look for the existing cached key under specific username.
+      const rsaKey = this.rsaKeyCache.get('oldTokenUser');
```

Comment:
This is nice I just thinking the `oldTokenUser` is a bit misleading. I think this is still a valid way to request a key. Maybe just call it `currentUser` could also probably look up the user id from the auth token but probably not necessary.

Reply:
True, will change this!